src/energy_comms/transform/epa.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+
+
+def transform(df):
```

Comment:
General note: I noticed that there are records where the zip code is null but there is latitude and longitude data. We could get the county associated with these lat, lon points so that we don't lose these records with null zip codes.

Reply:
good flag, thanks! i'll add that as an additional task for me